bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -24,13 +24,14 @@ metadata:
     containerImage: icr.io/cpopen/common-service-operator:latest
     createdAt: "2020-10-19T21:38:33Z"
     description: The IBM Common Service Operator is used to deploy IBM Common Services
-    nss.operator.ibm.com/managed-operators: cloud-native-postgresql,ibm-bts-operator
+    nss.operator.ibm.com/managed-operators: cloud-native-postgresql, ibm-bts-operator
```

Comment:
Additional space in string `cloud-native-postgresql,ibm-bts-operator` will cause NamespaceScope Operator failure